crawl-ref/source/dat/des/branches/coc.des

```diff
@@ -193,6 +193,548 @@ xxxxxxxxxxxxxxxxxxxxxxxxx.<xxxxxxxxxxxxxxx
 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 ENDMAP
 
+NAME:   nicolae_coc_bergin
+TAGS:   transparent no_pool_fixup
+DEPTH:  Coc:2-
+ORIENT: centre
+KFEAT:  0 = W
+KMONS:  0 = 0 w:12 / 9 w:3 / 8 w:1
+NSUBST: { = 1={ / .
+SUBST:  M = Wwww x:5 .:1, w = w:20 x:1, W = W 0:1
+MAP
+   wwM..@..Mw  wwM.@M
+  wwwwM...MwwwwwwM..ww
+ wwwwwwMWWwwwwwwwMWMww
+wwwwwwwwMWwwwwwwwMWwww
+wwwwwwwwMWwwwwwwMWMww
+MMwMMwwwwWWMwwwMWMww
+@.WWWWMMwwWwwwMWMwww
+MMwwwMWWWWWwwwWMwwwww
+wwwwMWWwwwWWwWWMwwwMww
+ wwwMWMwwM..WWwwMMMW..
+ wwwWMwwM.{{.wwwMWWW.@
+ wwWWWwwM.{{.WWWWMwMMM
+wwMWMwMwMM..WWMMMwwwww
+MMWWwwwMwWWMMWWwwwwww
+..WMwwwwWWMwwwWMwwww
+@.MwwwwwMWwwwwMWMwww
+MMwwwwwMWWwwwwwMWWww
+wwwwwwwMWMwwwwwwMWMww
+wwwwwMMWMwwwwwwwwWWMMM
+ wwwwMWWMwwwwwwwwMWW.M
+  wwwM...MwwwwwwwwWM..
+   wM..@.Mww     wM..@
+ENDMAP
+
+# Inspired by the bit from the Inferno where the sinners in the Caina region
+# of Cocytus are trapped in ice with only their heads sticking out. Granted,
+# these heads are a little more mobile than those ones.
+NAME:   nicolae_coc_caina
+TAGS:   transparent
+DEPTH:  Coc, !Coc:$
+ORIENT: centre
+KMONS:  1 = curse skull
+SUBST:  x = 1:5 xx.
+MAP
+     ..x.x
+   .x.......
+  ....x...x..x
+ ..x....x.....
+ x...x......x.
+.......x.x....
+x.x..x.........
+........}..x.x.
+.x...x.........
+..........x..x
+..x....x......
+x...x.....x..
+  x....x....x
+  ..x.....x.
+    ..x..
+ENDMAP
+
+NAME:   nicolae_coc_caroushell
+TAGS:   transparent no_pool_fixup no_monster_gen
+DEPTH:  Coc
+ORIENT: centre
+KMONS:  1 = diamond obelisk
```

Comment:
I figured it would work okay in Hell, since players are recommended to have a lot of AC in the Hells and it usually only takes a turn or two to get to the next "stop" on Mr. Polar Vortex's Wild Ride. Also, they're in Hell, so, you know, they knew what they signed on for.

I can downweight it for sure, though.

Reply:
I think it's ok at weight 1 or 2. Definitely `cool`.